platform/search/bundle.json

```diff
@@ -0,0 +1,34 @@
+{
+    "name": "Search",
+    "description": "Allows the user to search through the file tree.",
+    "extensions": {
+        "constants": [
+            {
+                "key": "GENERIC_SEARCH_ROOTS",
```

Comment:
Ah, I see - I think there's a misunderstanding of what "ROOT" is. "ROOT" is an unseen domain object that contains Packets, My Items, and anything else from the `roots` extension category. ("ROOT" is composed of `roots[]`)